daml-lf/validation/src/main/scala/com/digitalasset/daml/lf/validation/Typing.scala

```diff
@@ -727,35 +727,21 @@ private[validation] object Typing {
         checkExpr(exp, TScenario(typ))
     }
 
-    private def checkAnyType(typ: Type): Unit = {
-      // we check that typ contains neither variables nor quantifiers
-      TypeTraversable(typ)
+    // we check that typ contains neither variables nor quantifiers
+    private def checkSimpleType_(typ: Type): Unit = {
```

Comment:
Should this also be called `checkGroundType` to be consistent with https://github.com/digital-asset/daml/pull/3326#discussion_r342036768?

Reply:
fixed.